chia/full_node/full_node.py

```diff
@@ -473,7 +473,7 @@ async def _handle_one_transaction(self, entry: TransactionQueueEntry) -> None:
         peer = entry.peer
         try:
             inc_status, err = await self.add_transaction(entry.transaction, entry.spend_name, peer, entry.test)
-            entry.done.set_result((inc_status, err))
+            entry.done.set((inc_status, err))
```

Comment:
`done` is a `Future`, right? I can't see `set()` documented as a member function